client_src/js/utils/window.util.js

```diff
@@ -0,0 +1,8 @@
+export function reloadCurrentWindow() {
+  if (location.href.includes("submitEnvironment")) {
+    const hostName = window.location.protocol + "//" + window.location.host + "";
```

Comment:
Not sure on the submitEnvironment, it's old code I've forgotten about and need to dig into further. I believe it's something to do with the temp/humid/IAQ/Pressure. I left it currently just in case of that.